ui/imports/shared/views/chat/ChatContextMenuView.qml

```diff
@@ -127,7 +128,7 @@ StatusMenu {
         objectName: "chatFetchMessagesMenuItem"
         text: qsTr("Fetch messages")
         icon.name: "download"
-        enabled: !production
+        enabled: root.showDebugOptions
```

Comment:
np: should not `showDebugOptions` be defaulted with `production` value?

Reply:
It's kinda ok in both ways to me. Don't want to run another CI circle for this 😄